src/main/java/com/google/devtools/build/skydoc/SkydocMain.java

```diff
@@ -435,16 +433,17 @@ private Module recursiveEval(
     // process loads
     Map<String, Module> imports = new HashMap<>();
     for (String load : prog.getLoads()) {
-      Label relativeLabel = label.getRelativeWithRemapping(load, ImmutableMap.of());
+      Label apparentLabel = label.getRelativeWithRemapping(load, ImmutableMap.of());
```

Comment:
There's a desire to remove a bunch of methods in `Label` and `getRelativeWithRemapping` is one of them.

I think it's possible to use `Label.parseWithPackageContext`. Using it, leads to different code structure. But that structure might be even simpler.

Reply:
I am using `parseWithPackageContext` now. It didn't result in any structural changes though, am I missing some other change you had in mind?

Reply:
I was just guessing the recursion would rearrange differently. Looks ok.